app/models/event_stream.rb

```diff
@@ -32,4 +32,48 @@ def emit_notifications
   rescue => err
     _log.log_backtrace(err)
   end
+
+  #
+  # Purging methods
+  #
+
+  def self.keep_ems_events
+    VMDB::Config.new("vmdb").config.fetch_path(:ems_events, :history, :keep_ems_events)
```

Comment:
Yea, for some reason, I want to keep this the way it is. the world probably thinks of it as ems event. though, I defer to @lfu